docs/gettingStarted/QUICK_START.md

```diff
@@ -0,0 +1,152 @@
+    ],
+    "projects": [
+        "5e53a7928b7751764c16eee5"
+    ],
+    "components": [
+        {
+            "path": "src/components/button/Button.jsx",
+            "zeplinIds": [
+                "5dd41717b4eaa04034df4c6f"
+            ]
+        }
+    ]
+}
+```
+
+- `projects` and `styleguides` keys are the identifiers of projects and styleguides we'll connect components from.
+- `plugins` contain the module name of the plugins that the CLI uses to process our components. Each plugin may have its own custom configuration under `config` key.
+- `components` are the component files in our codebase. `zeplinIds` are the component IDs in Zeplin that the component connect to. You can alternatively use the `zeplinNames` key (instead of `zeplinIds`) to connect components using their names in Zeplin.
+
+Check [CONFIGURATION_FILE.md](../CONFIGURATION_FILE.md) for more details.
+
+## Update Connected Components
+
+Executing `zeplin connect` (or `npm run zeplin-connect` if Zeplin CLI packages are installed locally) will process the configuration/component files again, and update our Connected Components in Zeplin.
+
+You can add a task into your CI/CD pipeline to update Connected Components every time your component source code changes.
+
+## Connecting more components
+
+After initializing Connected Components, you can use the `zeplin connect add-components` (or `npm run zeplin-connect add-components` if Zeplin CLI packages are installed locally) command to connect more components to Zeplin interactively.
+
+It is possible to add components from other Zeplin projects or styleguides into the same configuration file. If you're using [Global Styleguides](https://blog.zeplin.io/announcing-global-styleguides-connecting-design-systems-to-engineering-65ad22bd0076), adding your components using the IDs of styleguide(s) will be enough. Connected Components will show up on parent Zeplin styleguide(s) as well.
+
+### Using CLI
+
+Open Zeplin and select another component that is not connected yet. Click on the _Connect to Code_ button again. This time, Zeplin will display the command to add this component to the existing Connected Component configuration. (Notice that the only difference is the `add-components` parameter, instead of `initialize`.)
+
+Similary, paste and execute the command. This component will now be added to the configuration file as well.
+
+```sh
+npx @zeplin/cli connect add-components \
+    --project-id 5e53a7928b7751764c16eee5 \
+    --component-id 5dd417142dff493fc295dce5
+```
+
+Rinse and repeat until you're done connecting all the component you need to!
+
+_Using `initialize` command will fallback to `add-components` if you have an existing `.zeplin/components.json` and vice versa._
+
+### Using VS Code extension
+
+You can also use the [Zeplin Visual Studio Code extension](https://zpl.io/vscode-extension) to manage the `.zeplin/components.json` file. It simplifies adding more and managing components in the configuration file.
```

Comment:
I think, we should use capital "E" here for "extension".

Same comment for the other ".md" files.